### PR TITLE
[ZEPPELIN-5762] Fix default profile activation option to build succes…

### PR DESCRIPTION
**Pull request by dntjr8096**
### What is this PR for?
When building from source code, if the Profile option is not given to the maven command by default, an error like Screenshot occurs. So I changed the default profile activation option in the "zeppelin-server module" to hadoop2.

### What type of PR is it?
Bug Fix

### Todos
* [ ] - Documentation?

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-5762

### How should this be tested?
I tested with Travis CI, which is set by default.

### Screenshots (if appropriate)
* ```./mvnw clean package -DskipTests ``` before modifying the profile with 'hadoop2'
image

* ```./mvnw clean package -DskipTests ``` after modifying the profile with 'hadoop2'
image

### Questions:
* Does the licenses files need to update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? Yes

### CODE REVIEW

File: zeppelin-server/pom.xml [hunk 1]
```diff
@@ -457,9 +457,6 @@
   <profiles>
     <profile>
       <id>using-source-tree</id>
-      <activation>
```
Comment (jongyoul) [inline zeppelin-server/pom.xml:460]: This is also needed. Please revert this change.

**Comment by jongyoul** [comment]
Overall, LGTM except for the minor comment.

File: zeppelin-server/pom.xml (hunk 1 above)
Comment (dntjr8096) [reply zeppelin-server/pom.xml:460]: Yes. Instead of deleting the line, I changed the activeByDefulat value to false.

Comment (dntjr8096) [reply zeppelin-server/pom.xml:460]: Yes. Instead of deleting the line, I changed the activeByDefulat value to false.

Comment (jongyoul) [reply zeppelin-server/pom.xml:460]: Yes, that should be reasonable but I think it's better not to change an unrelated thing for the PR so how about making another PR to handle this activation change?

Comment (dntjr8096) [reply zeppelin-server/pom.xml:460]: ok i'll do that

**Review by YeonghyeonKO — COMMENTED** [review]
LGTM

**Comment by jongyoul** [comment]
@YeonghyeonKO FYI, It's already been merged by #4403
